### PR TITLE
feat: add an invalid case to validate PR

### PR DESCRIPTION
**Pull request by pavel-mikhadyuk-redshelf**
We shouldn't put ":" after "fix" word if the jira ticket follows after the semantics prefix

### CODE REVIEW

File: .github/actions/validate-pr/validate_pr_test.sh [hunk 1]
```diff
@@ -204,3 +204,16 @@ ACTUAL="$($SCRIPT_DIR/validate_pr.sh)"
...
+# GIVEN
+export PR_BRANCH="fix-PAR-2061-remove-unnecessary-message-types"
+export TARGET_BRANCH="develop"
+export PR_TITLE="fix(PAR-2061): Remove unnecessary message types"
+
+# WHEN
+ACTUAL="$($SCRIPT_DIR/validate_pr.sh)"
+
+# THEN
+expect "$?" "1"
```
Comment (robaone-redshelf) [inline .github/actions/validate-pr/validate_pr_test.sh:219]: I was expecting a different test
https://redshelf.slack.com/archives/C0497KEL76J/p1734527057537459?thread_ts=1734522588.524649&cid=C0497KEL76J

File: .github/actions/validate-pr/validate_pr.sh [hunk 2]
```diff
@@ -39,6 +40,12 @@ if [[ "$TARGET_BRANCH" == "develop" ]] || [[ "$TARGET_BRANCH" =~ ^release/v ]] |
     echo "PR title must contain a valid Jira ticket ID (e.g., ABC-123)."
     exit 1
   fi
+
+  if [[ "$PR_TITLE" =~ $NEGATIVE_COLON_PATTERN ]]; then
+    echo "PR title contain a colon on an invalid position"
```
Comment (benkn) [inline .github/actions/validate-pr/validate_pr.sh:45]:
```suggestion
    echo "PR title contains a colon in an invalid position"
```

File: .github/actions/validate-pr/validate_pr_test.sh [hunk 3]
```diff
@@ -204,3 +204,31 @@ ACTUAL="$($SCRIPT_DIR/validate_pr.sh)"
...
+# THEN
+expect "$?" "0"
+
+echo Scenario: Invalid pr title because of colon
+beforeEach
+
+# GIVEN
+export PR_BRANCH="fix-PAR-2061-remove-unnecessary-message-types"
+export TARGET_BRANCH="develop"
+export PR_TITLE="fix:(PAR-2061) Remove unnecessary message types"
```
Comment (robaone-redshelf) [inline .github/actions/validate-pr/validate_pr_test.sh:227]: This is a valid pr title.

File: .github/actions/validate-pr/validate_pr.sh [hunk 4]
```diff
@@ -21,7 +21,7 @@ if [ "$TARGET_BRANCH" == "" ]; then
   exit 1
 fi
 
-SEMANTIC_PREFIXES="^(feat|fix|chore|docs|style|refactor|perf|test):"
+SEMANTIC_PREFIXES="^(feat|fix|chore|docs|style|refactor|perf|test)"
```
Comment (robaone-redshelf) [inline .github/actions/validate-pr/validate_pr.sh]: The correct pattern is `^(feat|fix|chore|docs|style|refactor|perf|test)[(:]`

**Review by robaone-redshelf — APPROVED** [review]

**Pull request merged**